csug/libraries.stex

```diff
@@ -998,6 +998,35 @@ When the new parameter \scheme{import-notify} is set to a true value,
 searches for the file containing each library it needs to load.
 The default value of this parameter is \scheme{#f}.
 
+%----------------------------------------------------------------------------
+\entryheader
+\formdef{library-search-handler}{\categorythreadparameter}{library-search-handler}
+\listlibraries
+\endentryheader
+
+When searching for library source or object code, \scheme{import},
+\scheme{compile-whole-program}, and \scheme{compile-whole-library}
+use the value of the \scheme{library-search-handler} parameter.
+This parameter must be set to a procedure that accepts four arguments:
+a symbol that provides context in \scheme{import-notify} messages,
+a list naming the desired library,
```

Comment:
I wonder if we should just call this a library name.  I noticed that is the language we use in Section 10.3 in CSUG (and 10.2 in TSPL).